src/main/java/step2/domain/Lotto.java

```diff
@@ -0,0 +1,29 @@
+package step2.domain;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+
+import static java.util.stream.Collectors.*;
+
+public final class Lotto<T> {
+    private List<LottoNumbers<T>> lotto = new ArrayList<>();
+
+    public List<LottoNumbers<T>> getLotto() {
+        return lotto;
```

Comment:
Collection의 참조를 바로 노출하면 외부에서 add 등의 메소드 호출을 통해 수정할 우려가 있어 보입니다.

아래와 같이 반환하면 어떨까요?

```suggestion
        return Collections.unmodifiablelist(lotto);
```